src/umockdev-pcap.vala

```diff
@@ -216,10 +216,10 @@ internal class IoctlUsbPcapHandler : IoctlBase {
 
```

Comment:
Hm, there is the `assert(cur_hdr.caplen)`. I guess that also needs a `uint32.from_little_endian`, but I am not entirely sure right now. It isn't going to cause issues though and we just rely on the USB header later on anyway.